src/main/java/com/carleton/CapstoneSystem/Prof.java

```diff
@@ -0,0 +1,71 @@
+package com.carleton.CapstoneSystem;
+
+import javax.persistence.Entity;
+import javax.persistence.Inheritance;
+import javax.persistence.OneToMany;
+import javax.rmi.PortableRemoteObject;
+import java.util.ArrayList;
+
+import java.util.HashMap;
+import java.util.List;
+
+@Entity
+@Inheritance
+public class Prof extends User {
+
+    private HashMap<String, Project> projectsSupervised;
+    @OneToMany
+    private List<Time> availableTimes;

```

Comment:
why does a prof have available times?

Reply:
for the presentation, a prof needs to give when they are available.